Dockerfile

```
@@ -15,9 +19,9 @@ ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
 
-RUN echo "APT::Install-Recommends \"false\";" | tee -a /etc/apt/apt.conf.d/defaults.conf
-RUN echo "APT::Get::Upgrade \"false\";" | tee -a /etc/apt/apt.conf.d/defaults.conf
-RUN echo "APT::Get::Install-Suggests \"false\";" | tee -a /etc/apt/apt.conf.d/defaults.conf
+RUN echo "APT::Install-Recommends \"false\";" | tee -a /etc/apt/apt.conf.d/buildpack.conf
+RUN echo "APT::Get::Upgrade \"false\";" | tee -a /etc/apt/apt.conf.d/buildpack.conf
+RUN echo "APT::Get::Install-Suggests \"false\";" | tee -a /etc/apt/apt.conf.d/buildpack.conf
```

```suggestion
RUN echo "APT::Install-Recommends \"false\";" | tee -a /etc/apt/apt.conf.d/buildpack.conf; \
         echo "APT::Get::Upgrade \"false\";" | tee -a /etc/apt/apt.conf.d/buildpack.conf; \
         echo "APT::Get::Install-Suggests \"false\";" | tee -a /etc/apt/apt.conf.d/buildpack.conf
```
Otherwise we get 3 layer with a single file